tests/stable_api_test.py

```diff
@@ -87,6 +87,10 @@ def test_stable_api_docstrings() -> None:
         nw_doc = getattr(nw, item).__doc__
         if item == "from_native":
             v1_doc = v1_doc.replace("native_dataframe", "native_object")
+        if item in {"to_py_scalar"}:
+            # We don't overwrite the docstring for these
```

Comment:
Just so I know what to do/review in the future, which kind of methods should be added here?

Reply:
I think all of them to be honest, it's probably simpler to just use `import narwhals as nw` in all docstrings and forget about this test 😅 i'll make a separate pr